src/components/errorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import LinkButton from '../linkButton/LinkButton';
+import { LinkType } from '../../../studio/lib/payloads/navigation';
+import Text from 'src/components/text/Text';
+import { ReactNode } from 'react';
+import styles from './errorMessage.module.css';
+import Button from '../buttons/Button';
+
+
+export const ErrorMessage = ({ title, children }: { title: string, children: ReactNode }) => {
+  return (
+    <section className={styles.wrapper}>
+      <div className={styles.error}>
+        <Text type="h3">Error - {title}</Text>
+        {children}
+      </div>
+    </section>
+  );
+};
+
+
+const studioLink = {
+  _key: 'go-to-sanity-studio',
+  _type: 'link',
+  linkTitle: 'Go to Studio',
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: 'studio',
+  },
+};
+
+
+export const MissingContentErrorMessage = ({ description }: { description: string }) => {
```

Comment:
Hm, this is a good setup but it's also easy to get confused as the current structure groups components into unique folders:  errorMessage/ErrorMessage. 

So when seeing MissingContentErrorMessage I'd expect it to be in: missingContentErrorMessage/MissingContentErrorMessage

What we can do is set ErrorMessage with a type og better yet make it more universal like:

```
const ErrorMessage = ({ onClick }: { onClick?: () => void }) => {
  return (
    <section className={styles.wrapper}>
      <div className={styles.error}>
        <Text type="h3">{title}</Text>
        <Text>{description}</Text>
        <div className={styles.buttonWrapper}>
          {onClick && (
            <Button type="secondary" onClick={onClick}>
              Try again
            </Button>
          )}
          <LinkButton type="secondary" link={link} />
        </div>
      </div>
    </section>
  );
};
```

that way we make sure that error messages are universal in it's content and only have button as optional

Reply:
So the "Return to home" link should be present in all cases?

Reply:
But we can definitely move them into separate folder if that's cleaner

<img width="360" alt="image" src="https://github.com/user-attachments/assets/abe0386b-efce-4451-a973-5e8f2ae3cf92">

Reply:
Also, having a `description` only as a string (instead of allowing a children node) will limit what you can show in the error message. That migh be fine depending on the use cases.

Reply:
you can set link as optional also if that's better; but for cases with !landingID I'd set the link as Go to Studio and then path /studio

Reply:
Updated now to include optional link and tryAgain callback in generic component